### DISCUSSION TITLE
Colab @tvm.script.ir_module报错

### DISCUSSION THREAD

**Discussion by Eddiewdy** [mlc-ai/mlc-zh#167] [Q&A]
在tvm script中使用@R.function就会产生报错，如果不使用relax就可以正常运行script。环境是colab + mlc-ai-nightly。
![image](https://user-images.githubusercontent.com/50788764/203685608-1d50b560-0d98-48bb-9de0-0020a335fee9.png)
![Xnip2022-11-24_11-10-57](https://user-images.githubusercontent.com/50788764/203685672-42ea6c03-c465-41d0-9a12-8294868ce118.jpg)
![image](https://user-images.githubusercontent.com/50788764/203685709-f5aa2a31-7f09-4629-ab1c-db12bd88d374.png)

**Comment by Jerrrrry-Zhu** [comment]
我也碰到同样的问题，请问你解决了吗

**Reply by Eddiewdy** [reply]
试试运行下面的指令
`!wget https://github.com/mlc-ai/utils/releases/download/v0.9.dev0/mlc_ai_nightly_cu110-0.9.dev2227%2Bg06c0ef3be-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl`
`!pip install ./mlc_ai_nightly_cu110-0.9.dev2227+g06c0ef3be-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl`

**Comment by Hzfengsy** [comment]
@Andy-muse @Eddiewdy 目前Relax本身正在更新Script API，短期内notebook可能会出现运行错误。等API更新完成后将会更新notebook

**Reply by elvissszhang** [reply]
@Hzfengsy 这个问题解决了么，遇到同样的问题
